gensim/models/tfword2vec.py

```diff
@@ -0,0 +1,206 @@
+import tensorflow as tf
+from tensorflow.models.embedding.word2vec_optimized import Word2Vec
+from gensim.models.word2vec import Word2Vec as GensimWord2Vec, Vocab
+from gensim import utils
+from six import string_types
+import logging
+logging.basicConfig(format='%(asctime)s : %(levelname)s : %(message)s', level=logging.INFO)
```

Comment:
This doesn't belong in module scope -- libraries do not set up logging. That's up to applications that use them.